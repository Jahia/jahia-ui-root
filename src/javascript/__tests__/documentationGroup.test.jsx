import React from 'react';
import renderer from 'react-test-renderer';
import {DocumentationGroup} from '../Jahia/jahia.routes.jsx';

describe('documentation group', () => {

    test('links', () => {
        window.contextJsParameters = { links: {whatsNew: "mock_url", documentation: "mock_url"}};
        const component = renderer.create(
            <DocumentationGroup />,
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
});
