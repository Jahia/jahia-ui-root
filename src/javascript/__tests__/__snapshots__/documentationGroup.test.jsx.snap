// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`documentation group links 1`] = `
Array [
  <li
    className="_38IL5xx6MzHEv52EctnPV_"
  >
    <a
      className="_38IL5xx6MzHEv52EctnPV_ _1HF_Uc-LbdRZywnGmFDG7k"
      href="mock_url"
      rel="noopener noreferrer"
      target="_blank"
    >
      <div
        className="_3Wbxt6soOkNE807wmyu_iI"
      >
        <div
          className="_1Ai3EiPwY-pZNUwN6mrrmu"
        />
        <div
          className="_3ejht2paDjYw7X613AENsu"
        >
          <span
            className="_2UHrI1QWqxLhuY_tZNdPZo _1XC6NkjLjDAWYIpUQk5eHx _1Bzmq62UqnegYx7zkkgnjt _3McV6_vYIoTTSdX0ayC7p5"
          >
            jahia-ui-root:primaryNavigation.documentation.newAtJahia
          </span>
        </div>
      </div>
    </a>
  </li>,
  <li
    className="_38IL5xx6MzHEv52EctnPV_"
  >
    <a
      className="_38IL5xx6MzHEv52EctnPV_ _1HF_Uc-LbdRZywnGmFDG7k"
      href="mock_url"
      rel="noopener noreferrer"
      target="_blank"
    >
      <div
        className="_3Wbxt6soOkNE807wmyu_iI"
      >
        <div
          className="_1Ai3EiPwY-pZNUwN6mrrmu"
        />
        <div
          className="_3ejht2paDjYw7X613AENsu"
        >
          <span
            className="_2UHrI1QWqxLhuY_tZNdPZo _1XC6NkjLjDAWYIpUQk5eHx _1Bzmq62UqnegYx7zkkgnjt _3McV6_vYIoTTSdX0ayC7p5"
          >
            jahia-ui-root:primaryNavigation.documentation.label
          </span>
        </div>
      </div>
    </a>
  </li>,
]
`;
